FALSEHOODS.md

```diff
@@ -118,3 +118,8 @@ surprising how many false assumptions programmers continue to make about them.
 
     In Egypt, it is common for phone numbers to be written in native digits.
 
+1.  **Phone numbers have only one prefix**
+
+    In the mid-90s in Iceland, phone numbers changed from 5 and 6 digits to 7 digits.
```

Comment:
I'd start off saying "It is common during transition periods for countries to support both old and new ways of dialling a phone number." And then you can add the Iceland example.